src/controllers/user.js

```diff
@@ -32,24 +32,28 @@ export const create = async (req, res) => {
 
 export const getById = async (req, res) => {
   const id = parseInt(req.params.id)
+  const { role } = req.user
 
   try {
-    const foundUser = await User.findById(id)
-
+    let foundUser = {}
+    const isTeacher = 'TEACHER'
+    if (role === isTeacher) {
```

Comment:
isTeacher should be `role === 'TEACHER'` then if the if statement will become `if (isTeacher)`